conformance_tests/sysman/test_sysman_memory/src/test_sysman_memory.cpp

```diff
@@ -443,4 +462,91 @@ TEST_F(
     memoryThread.join();
   }
 }
+
+void getMemoryStateTemp(ze_device_handle_t device) {
```

Comment:
function names should be in snake case.

Reply:
yeah, but the other functions in test_sysman_memory were in camel case. Thanks! will change

Reply:
Changed the functions names I added to snake_case.